src/components/Accordion/intefaces/AccordionItemInterface.ts

```diff
@@ -0,0 +1,7 @@
+import { ComponentInterface } from '@paper/core/interfaces/ComponentInterface';
+
+export interface AccordionItemInterface extends ComponentInterface {
+  onOpen?: Function;
+  eventKey?: number | any | string;
```

Comment:
If you use any, the number and string are not needed.

```suggestion
  eventKey: any;
```